packages/transactions/src/clarity/types/principalCV.ts

```diff
@@ -51,8 +49,8 @@ function principalCV(principal: string): PrincipalCV {
  *  // { type: 5, address: { type: 0, version: 22, hash160: 'a5d9d331000f5b79578ce56bd157f29a9056f0d6' } }
  * ```
  *
- * @visit
- * {@link https://github.com/hirosystems/stacks.js/blob/master/packages/transactions/tests/clarity.test.ts clarity test cases for more examples}
+ * @see
```

Comment:
Is `see` an alias for `visit`? I see that other modified files (optionalCV, responseCV, ...) still uses `visit`

Reply:
Thanks I'll change all of them -- yeah, I think `@visit` technically doesn't exist in tsdoc